server/data/orchestrationApiClient.test.ts

```diff
@@ -36,15 +36,17 @@ describe('orchestrationApiClient', () => {
   describe('bookVisit', () => {
     it('should book a visit from an application', async () => {
       const applicationReference = 'aaa-bbb-ccc'
+      const requestBody: BookingOrchestrationRequestDto = {
+        applicationMethodType: 'WEBSITE',
+        allowOverBooking: false,
+        actionedBy: bookerReference.value,
+        userType: 'PUBLIC',
+      }
 
       const result = { reference: 'ab-cd-ef-gh' } as Partial<VisitDto>
 
       fakeOrchestrationApi
-        .put(`/visits/${applicationReference}/book`, <BookingOrchestrationRequestDto>{
-          applicationMethodType: 'WEBSITE',
-          allowOverBooking: false,
-          actionedBy: bookerReference.value,
-        })
+        .put(`/visits/${applicationReference}/book`, requestBody)
```

Comment:
I've refactored this (and others in the file similarly) so that `requestBody` can be explicitly typed. Previously, using type casting like `<BookingRequestDto>{ obj }` wouldn't flag up if a new required property was missing.